### PR TITLE
feat: added createdAt information

### PR DESCRIPTION
**Pull request by CzarekDryl**
## Testing

* Create any action using permissions

**New stuff** ✨

* Changed box description
* Added information when action was created
image

Resolves [2310](https://github.com/JoinColony/colonyCDapp/issues/2310)

### CODE REVIEW

**Review by melyndav — DISMISSED** [review]
Looks great @CzarekDryl thanks for the update to this component. Nice work!

**Review by arrenv — DISMISSED** [review]
Thank you @CzarekDryl, looks good to me :)

File: src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/utils.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { format, isToday, isYesterday } from 'date-fns';
+
+export const formatDate = (createdAt: string | undefined) => {
```
Comment (rdig) [inline src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/utils.ts:3]: There's no need for this util as `react-intl` already exports a `FormattedDate` component which you can use directly, and already contains all the formatting options that you might need.

It also exports `FormattedDateParts` which gives you as much granular control as you'd want.

We already use it here: https://github.com/JoinColony/colonyCDapp/blob/feat/15899525-created/src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakeItem.tsx#L28

**Review by rdig — CHANGES_REQUESTED** [review]
The change itself is good, but please don't add redundant logic, when we already have ways of dealing with date and time formatting.

File: src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/utils.ts (hunk 1 above)
Comment (CzarekDryl) [reply src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/utils.ts]: @rdig I wasn't sure how FormattedDateParts works. Take a look to see if this is possible.

**Review by melyndav — APPROVED** [review]
Re-approving this one @CzarekDryl

**Review by rdig — APPROVED** [review]
All good now. 

Would have personally not had created the extra `formatData` fn, and just rendered everything in component directly, but It can go in as it is.

**Pull request merged**